index.js

```diff
@@ -25,6 +25,9 @@ function mewt(target) {
       return mewt(newObj)
     },
     $unset: prop => {
+      if (isA && Number.isInteger(prop)) {
+        return mewt([...target.slice(0, prop), ...target.slice(prop + 1)])
```

Comment:
Ah yes, of course. My bad! Filter looks good but might be expensive on large arrays?